src/session/session.rs

```diff
@@ -432,6 +444,24 @@ impl<const LEN_SIZE: usize>
         )
     }
 
+    fn with_connect_tcp_encrypted(
```

Comment:
Can we re-name this method to `with_tcp_encrypted`? The current name is not very much readable

Reply:
I hope this is what you wanted: https://github.com/Internet2-WG/rust-internet2/pull/30 . If not feel free to break my code  :P

Reply:
Thank you!